src/main/java/gtPlusPlus/xmod/gregtech/api/metatileentity/implementations/base/GregtechMeta_MultiBlockBase.java

```diff
@@ -489,24 +486,17 @@ public int canBufferOutputs(ItemStack[] aItemOutputs, FluidStack[] aFluidOutputs
 				}
 				else {
 					int aStackSize = aY.stackSize * aParallelRecipes;
-					if (aStackSize > 64) {
-						int aSlotsNeedsForThisStack = (int) Math.ceil((double) ((float) aStackSize / 64f)); 
-						// Should round up and add as many stacks as required nicely.
-						aRecipeSlotsRequired += aSlotsNeedsForThisStack;					
-						for (int o=0;o<aRecipeSlotsRequired;o++) {
-							int aStackToRemove = (aStackSize -= 64) > 64 ? 64 : aStackSize;		
-							aY = aY.copy();
-							aY.stackSize = 0;
-							aInputMap.add(new FlexiblePair<ItemStack, Integer>(aY, aStackToRemove));
-						}					
-					}
-					else {
-						// Only requires one slot
-						aRecipeSlotsRequired++;
+
+					int aSlotsNeedsForThisStack = (int) Math.ceil((float) aStackSize / aY.getMaxStackSize());
+					// Should round up and add as many stacks as required nicely.
+					for (int o=0;o<aSlotsNeedsForThisStack;o++) {
+						if (aStackSize < 1) break;
+						int aStackToRemove = Math.min(aStackSize, aY.getMaxStackSize());
+						aStackSize -= aY.getMaxStackSize();
```

Comment:
Shouldn't cause issues but shouldn't you decrease it by `aStackToRemove ` instead of the max stack size?

Reply:
Oh yes true